lms/djangoapps/course_api/blocks/serializers.py

```diff
@@ -44,7 +45,7 @@ def to_representation(self, block_key):
             ),
         }
 
-        if 'lti_url' in self.context['requested_fields']:
+        if settings.FEATURES.get("ENABLE_LTI_PROVIDER") and 'lti_url' in self.context['requested_fields']:
```

Comment:
Oh yeah.  This LTI provider change is outside the scope of this ticket.  I included it in here since it was resulting in a 500 error on prod when the `lti_url` field is passed in but the feature flag is disabled.